cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -685,7 +681,7 @@ pub(crate) fn define(
     let iflags: &TypeVar = &ValueType::Special(types::Flag::IFlags.into()).into();
     let fflags: &TypeVar = &ValueType::Special(types::Flag::FFlags.into()).into();
 
-    let b1: &TypeVar = &ValueType::from(LaneType::from(types::Bool::B1)).into();
+    let b1: &TypeVar = &ValueType::from(LaneType::from(types::Int::I8)).into();
```

Comment:
Do we still need a typevar named `b1` at all? Is this used for "truthy" things like icmp return values etc? If so, perhaps we could call it `truthy` or somesuch?

Reply:
I renamed it to `i8`, as that should hopefully give good documentation for the places that it's used.